test/models/graphcast/utils.py

```diff
@@ -36,3 +37,13 @@ def get_icosphere_path():
     script_path = os.path.abspath(__file__)
     icosphere_path = os.path.join(os.path.dirname(script_path), "icospheres.json")
     return icosphere_path
+
+
+def compare_quantiles(t: torch.Tensor, ref: torch.Tensor, quantiles: List[float], tolerances: List[float]):
```

Comment:
This is a nice utility! Would be nice if you could add some docstrings so other tests can also use this.

Reply:
It also would be great to have a place for these, actually.